src/port/ui/ContentBrowser.cpp

```diff
@@ -79,6 +88,23 @@ namespace Editor {
         }
     }
 
+    // For C-actors
+    std::unordered_map<std::string, std::function<void(const FVector&)>> CActorList = {
+        { "Item Box", [](const FVector& pos) {
+              Vec3f position = {pos.x, pos.y, pos.z};
+              Vec3s rot = {0, 0, 0};
+              Vec3f vel = {0, 0, 0};
+            s32 id = add_actor_to_empty_slot(position, rot, vel, 12); // item box
+            s32 height = spawn_actor_on_surface(position[0], position[1] + 10.0f, position[2]);
+
+            Actor* actor = CM_GetActor(id);
+            actor->unk_08 = height;
+            actor->velocity[0] = position[1];
+            actor->pos[1] = height - 20.0f;
+
+        }},
+    };
+
     std::unordered_map<std::string, std::function<AActor*(const FVector&)>> ActorList = {
```

Comment:
can be better to use the regestry no ?

Reply:
In the future on this one. It's temporary for now.